pkg/machines/libvirt.es6

```diff
@@ -357,16 +357,29 @@ LIBVIRT_PROVIDER = {
             });
         }
 
+        function forceFileDelete() {
+            if (options.storage) {
+                let rm = ['rm', '-f']; // "-f" to not fail on already missing files
+                options.storage.forEach(storageToDelete => {
+                    if (storageToDelete.file) {
+                        return rm.push(storageToDelete.file);
+                    }
+                });
+
+                return rm.length > 2 && cockpit.spawn(rm, { err: "message", environ: ['LC_ALL=C'] });
```

Comment:
Since you don't handle errors here, IMHO this should drop `err: message`, so that errors instead appear in the journal. There's also no need to force the locale.

Alternatively you can use `cockpit.file(path).replace(null)` which avoids calling a new process.